docs/articles/building-source.md

```diff
@@ -0,0 +1,8 @@
+# Building Files from source
+
+1. Install Microsoft Visual Studio and the UWP Development kit.
```

Comment:
Any version of Visual Studio?

Reply:
Wdym?

Reply:
Should it be 2019 and above? Or is VS 6 fine too (I doubt that but you get the idea)

Reply:
yes, should be VS2019

Reply:
and above

Reply:
nooo, yair just said it was 17 and above
```suggestion
1. Install Microsoft Visual Studio 2017 or later and the UWP Development kit.
```